RedMenu/RedMenuClient/menus/PlayerMenu.cs

```diff
@@ -26,6 +27,23 @@ public static void SetupMenu()
             menu.AddMenuItem(healBtn);
             menu.AddMenuItem(restoreStaminaBtn);
             menu.AddMenuItem(restoreEyeBtn);
+            menu.OnItemSelect += (_menu, _item, _index) =>
+            {
+                int index = (int)_index;
+                switch(index)
+                {
+                    case 0:
+                        Function.Call<int>((Hash)0xC6258F41D86676E0, PlayerPedId(), (int)0, (int)100);
```

Comment:
Also I'm pretty sure that there is a native to _get_ the max health/dead eye/stamina values for the player or ped. Please try to implement those because that way if another resource modifies the max health, this resource will not fuck that up and it'll set it to the actual max health for this player/ped.

Reply:
Nevermind, this is the inner core stuff you're doing now. What I mentioned is for the outer cores.